src/policies/AllGetWithSelect.rego

```diff
@@ -0,0 +1,17 @@
+package AllGetWithSelect
+
+import future.keywords.if
+
+default allow := false
+
+allow 
+{
+    input.method == "GET"
+    input.query["select"] != null
```

Comment:
Do we need to specify the `$` in `$select`?

Reply:
On beta endpoints `$` is optional, I thought we could normalize without it. We can choose the normalization with `$` sign as well. I am OK with both.

https://learn.microsoft.com/en-us/graph/query-parameters?tabs=http

Reply:
addressed